ir_radon_zwart_powell.jl

```diff
@@ -0,0 +1,110 @@
+using LazyGrids: btime, @timeo # not exported; just for timing tests here
+using InteractiveUtils: versioninfo
+   # function output = ir_radon_zwart_powell(theta, yr)
+   #
+   # Compute analytic 2D Radon transform of Zwart-Powell box spline.
+   #
+   # in
+   # theta -	ray angle in radian
+   # rr	-distance between the point and the ray (normalized by the pixel size)
+   # output: radon transform of the Zwart-Powell box spline element
+   #
+   # This is the modified version of the code written by [1]
+   # to avoid symbolic math operation.
+   # Code written by Seongjin Yoon, Univ. of Michigan, Jan 2015
+   #
+   # Reference
+   # [1] A. Entezari, M. Nilchian, and M. Unser, "A box spline calculus
+   # for the discretization of computed tomography reconstruction problems,"
+   # IEEE Trans. Med. Imaging, vol. 31, no. 8, pp. 1532–1541, 2012.
+   #
+   # 2015-08-10 Jeff Fessler, added self test and parallelized
+    # if nargin < 2, ir_usage, return, end
+
+    function ir_radon_zwart_powell(theta, rr)
+    dim = size(theta)
+    theta = vec(theta) #converting the matrix theta into a vector with 18281 elements
+    
+    zeta = zeros(length(theta), 4) #Martrix zeta of dimensions: No. of rows = number of elements in Zeta, No. of coloumns = 4
+    zeta[:,1] = cos.(theta)
+    zeta[:,2] = sin.(theta)
+    #zeta[:,2] = sin.(theta); #in the theta vector the values of above 3.146 gives at for example index 362
+    # different values in matlab and julia.. this maybe a probably using the sin function
+    zeta[:,3] .= zeta[:,1] .+ zeta[:,2]
+    zeta[:,4] .= zeta[:,1] .- zeta[:,2] 
+    #cond = abs.(zeta) .>= eps(1.0)
+    cond = abs.(zeta) .>= 1.1920929e-07  #eps('single') matlab different than eps(1.0)- julia
+    N = sum(cond, dims = 2) #returns the sum of the row in the matrix
+    N = vec(N)      
+    output = BoxSp4(rr[:], zeta, cond, N) ./ factorial.(N.-1)
+    output = reshape(output, dim)
+    return output
+    end 

```

Comment:
```suggestion
 end 
```

function `end` should not be indented.  please follow the style in my other julia libraries.